.github/workflows/build-and-test.yaml

```diff
@@ -20,5 +20,5 @@ jobs:
       - name: Publish to MavenLocal
         run: ./gradlew publishToMavenLocal
 
-      - name: Run JVM tests and Debug Unit Tests
-        run: ./gradlew allTests
+      - name: Run tests
+        run: ./gradlew iosSimulatorArm64Test iosX64Test jvmTest testDebugUnitTest
```

Comment:
Because back-in-time-plugin automatically disables compiler extensions for the android release build, running `allTests` results in failure. To avoid this problem, specified each test manually.